src/MacroTools/QuestSystem/UtilityStructs/ObjectiveUnitIsDead.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using WCSharp.Events;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+
+namespace MacroTools.QuestSystem.UtilityStructs
+{
+  public class ObjectiveUnitIsDead : Objective
```

Comment:
Add `sealed` keyword IMO; not designed to be inherited.

Reply:
Done